src/main/webapp/app/entities/flair-bi/modal/modal-tabs/condition/condition.component.html

```diff
@@ -44,31 +52,37 @@
             <input type="text" class="form-control" ng-model="vm.condition.value" />
         </div>
-        <div ng-if="vm.condition['@type'] === 'Contains' || vm.condition['@type'] === 'NotContains'" class="col-md-5 field">
-            <tags-input-grpc placeholder="Add Values" use-strings="true" min-length="1" ng-model="vm.condition.values" replace-spaces-with-dashes="false">
-                <auto-complete-grpc match-class=" filter-autocomplete " source="vm.load($query, vm.condition.featureName)" min-length="0
+        <div ng-if="vm.condition['@type'] === 'Contains' || vm.condition['@type'] === 'NotContains'"
+            class="col-md-5 field">
+            <tags-input-grpc placeholder="Add Values" use-strings="true" min-length="1" ng-model="vm.condition.values"
+                replace-spaces-with-dashes="false">
+                <auto-complete-grpc match-class=" filter-autocomplete "
+                    source="vm.load($query, vm.condition.featureName)" min-length="0
                 " load-on-focus="true " load-on-empty="true " max-results-to-show="32 "></auto-complete-grpc>
             </tags-input-grpc>
         </div>
         <div class="col-md-2 field">
-            <div ng-show="vm.showAdd" class="data-constraint-operation-button add-button icon" ng-click="vm.addComposition()">
+            <div ng-show="vm.showAdd" class="data-constraint-operation-button add-button icon"
+                ng-click="vm.addComposition()">
                 <svg class="flair-add-icon flair-oprations-icon">
-                     <use xlink:href="#flair-add-icon" xmlns:xlink="http://www.w3.org/1999/xlink">
-                     </use>
+                    <use xlink:href="#flair-add-icon" xmlns:xlink="http://www.w3.org/1999/xlink">
+                    </use>
                 </svg>
             </div>
-            <div ng-show="vm.showDelete" class="data-constraint-operation-button remove-button icon" ng-click="vm.removeCondition()">
+            <div ng-show="vm.showDelete" class="data-constraint-operation-button remove-button icon"
+                ng-click="vm.removeCondition()">
                 <svg class="flair-delete-icon flair-oprations-icon">
-                     <use xlink:href="#flair-delete-icon" xmlns:xlink="http://www.w3.org/1999/xlink">
-                     </use>
+                    <use xlink:href="#flair-delete-icon" xmlns:xlink="http://www.w3.org/1999/xlink">
+                    </use>
                 </svg>
             </div>
         </div>
     </div>
 </div>
 
 <div class="condition-component-wrapper" ng-if="vm.condition['@type'] === 'Or' || vm.condition['@type'] === 'And'">
-    <condition-component show-add="true" show-delete="true" features="vm.features" condition="vm.condition.firstExpression" datasource-id="vm.datasourceId"></condition-component>
+    <condition-component show-add="true" show-delete="true" features="vm.features"
```

Comment:
if there is no change why this line is showing red...please change setting of ur editor...otherwise it is hard to indetify the changes